test/models/settings/email_setting_test.rb

```diff
@@ -22,6 +22,11 @@ class EmailSettingTest < ActiveSupport::TestCase
     assert_equal({"address" => "string@example.com", "authentication" => 'plain'}, Setting::Email.delivery_settings)
   end
 
+  test 'value of email_subject_prefix should not be more than 255 characters' do
+    Setting[:email_subject_prefix] = 'p' * 255
+    refute_valid Setting::Email.new
```

Comment:
I think the test now tests the wrong thing. This is probably invalid because no name is set. Can you please check if the setting you create above is not valid?